test/src/test/java/org/axonframework/test/saga/FixtureTest_RegisteringMethodEnhancements.java

```diff
@@ -46,12 +46,12 @@
 import static org.junit.jupiter.api.Assertions.*;
 
 /**
- * This test class is intended to test whether the registration of a
- * {@link HandlerDefinition} and a
+ * This test class is intended to test whether the registration of a {@link HandlerDefinition} and a
  * {@link HandlerEnhancerDefinition} go according to plan.
  *
  * @author Steven van Beelen
  */
+@Disabled("#3710 Reenable after ParameterResolver fix")
```

Comment:
Should be fixed once the old Saga uses the new UnitOfWork for migration purposes. Currently fails because the `TrackedEventMessage` is obsolete and the `TrackingToken` should come from the `ProcessingContext` insteand.